pkg/tsdbctl/tsdbctl.go

```diff
@@ -72,7 +72,7 @@ func NewRootCommandeer() *RootCommandeer {
 	cmd.PersistentFlags().StringVarP(&commandeer.v3ioPath, "server", "s", defaultV3ioServer,
 		"Web-gateway (web-APIs) service endpoint of an instance of\nthe Iguazio Continuous Data Platfrom, of the format\n\"<IP address>:<port number=8081>\". Examples: \"localhost:8081\"\n(when running on the target platform); \"192.168.1.100:8081\".")
 	cmd.PersistentFlags().StringVarP(&commandeer.cfgFilePath, "config", "g", "",
-		"Path to a YAML TSDB configuration file. When this flag isn't\nset, the CLI checks for a "+config.DefaultConfigurationFileName+" configuration file in the\ncurrent directory. CLI flags override file configurations.\nExample: \"~/cfg/my_v3io_tsdb_cfg.yaml\".")
+		"Path to a YAML TSDB configuration file. When this flag isn't\nset, the CLI checks for a "+config.DefaultConfigurationFileName+" configuration\nfile in the current directory. CLI flags override file\nconfiugrations. Example: \"~/cfg/my_v3io_tsdb_cfg.yaml\".")
```

Comment:
Consider using dashes instead of underscores for consistency with regular configuration file name

Reply:
Actually, because this is an example of what the user might use I think it's good to give an example that uses another naming convention.